File: platforms/WindowsPhone7/Accounter/Core/ResultList.cs
using System;
using System.Collections.Generic;

namespace Accounter.Core
{
	public class ResultList
	{

		private List<Record> _records = new List<Record>();

		private Boolean _isMultiSelectionEnabled;

		public String Title { get; set; }

		public List<Record> Records
		{
			get { return _records; }
			set { _records = value; }
		}

		public Boolean MultiSelectionEnable
		{
			get { return _isMultiSelectionEnabled; }
			set { _isMultiSelectionEnabled = value; }
		}

	}
}

File: platforms/Android/Accounter/src/com/vimukti/accounter/result/ResultList.java
package com.vimukti.accounter.result;

import java.util.ArrayList;
import java.util.List;

public class ResultList {
	private List<Record> records = new ArrayList<Record>();
	private boolean isMultiSelection;
	private String title;

	public void setMultiSelection(boolean isMultiSelection) {
		this.isMultiSelection = isMultiSelection;
	}

	public boolean isMultiSelection() {
		return isMultiSelection;
	}

	public void setRecords(List<Record> records) {
		this.records = records;
	}

	public List<Record> getRecords() {
		return records;
	}

	public void setTitle(String title) {
		this.title = title;
	}

	public String getTitle() {
		return title;
	}

}
